client/src/components/Header.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import Logo from './Logo';
+
+const Container = styled.div`
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: left;
+  padding-left: 10px;
+  width: 100%;
+  height: 10%;
+`;
+
+const Title = styled.h1`
+  font: 300 1.2rem 'Roboto', sans-serif;
+  color: #aeb2f5;
+  padding-top: 10px;
+  margin-left: 20px;
+`;
```

Comment:
```suggestion
const Title = styled.h1`
  font: 300 1.2rem 'Roboto', sans-serif;
  color: #aeb2f5;
  margin-left: 20px;
  display: flex;
  align-items: center;
`;
```

This will help to align your title with the logo image and LogOut button components.

Reply:
great simple fix. thanks! 👍  🎉